lib/main.ts

```diff
@@ -22,6 +22,11 @@ export interface TranspilerOptions {
    * reason for the failure. When falsey, semantic diagnostics will never be output.
    */
   semanticDiagnostics?: boolean;
+  /**
```

Comment:
how slow are we talking?

Reply:
It takes one minute to emit with the --skip-formatting flag and takes around 40 seconds to format using dartfmt. It takes around 7 minutes to emit and format without the flag.